pkg/vm/manager.go

```diff
@@ -333,6 +329,27 @@ func (m *Module) Run(vm pkg.VM) error {
 		return fmt.Errorf("a vm with same name already exists")
 	}
 
+	cfg := cloudinit.Configuration{
+		Metadata: cloudinit.Metadata{
+			InstanceID: vm.Name,
+			Hostname:   vm.KernelArgs["host"],
+		},
+		Extension: cloudinit.Extension{
+			Entrypoint:  vm.Entrypoint,
+			Environment: vm.Environment,
+		},
+	}
+
+	// TODO: user config should be added as another property on the
+	// VM workload data. For now we always use the SSH_KEY from the env
+	// if provided
+	if key, ok := vm.Environment["SSH_KEY"]; ok {
+		cfg.Users = append(cfg.Users, cloudinit.User{
+			Name: "root",
+			Keys: strings.Split(key, "\n"), // in case ssh_key container multiple keys. is this a usecase (?)
```

Comment:
I think `,` would make more sense as a separator?

Reply:
The reason i used `\n` instead is that what we did before with SSH_KEY is that we directly echoed that in authorized_keys. Hence the way to pass multiple keys was basically to put them as separate lines. I didn't want to break this in case someone is already using it like this.

Reply:
alright, not sure if we need it then but can we perhaps support both `\n` and `,` for now?